draganddrop/draganddrop/MainPage.xaml.cs

```diff
@@ -0,0 +1,62 @@
+﻿using System;
```

Comment:
This happened, for example, to the FeedReader sample. It had a customized default.js file that got overwritten when we deployed new JS templates. See f59dbfe63c5e0c3f73d5db8d70abd2fabd62ed08 which introduced the problem, 74d496d95c9f9a43a3bf3ca581bb8b8f0b78151c which fixed it, and c697adc1a7f3d2cc7aa5fabe83f0a73e35078fd7 which renamed default.js to main.js so it won't get inadvertently stomped by future template changes.